lane.py

```diff
@@ -1503,7 +1542,7 @@ def works_for_specific_ids(self, _db, work_ids, work_model=mw):
 
         b = time.time()
         logging.info(
-            u"Obtained %s×%d in %.2fsec", work_model.__name__, len(results), b-a
+            u"Obtained %sx%d in %.2fsec", work_model.__name__, len(results), b-a
```

Comment:
This is an unrelated change. The × character displays as `\u00d7` when the log format is JSON, which is more annoying than seeing the letter x.